vector/src/main/java/im/vector/app/features/call/webrtc/ScreenCaptureServiceConnection.kt

```diff
@@ -27,10 +27,17 @@ class ScreenCaptureServiceConnection @Inject constructor(
         private val context: Context
 ) : ServiceConnection {
 
+    interface Callback {
+        fun onServiceConnected()
+    }
+
     private var isBound = false
     private var screenCaptureService: ScreenCaptureService? = null
+    private var callback: Callback? = null
+
+    fun bind(callback: Callback) {
+        this.callback = callback
```

Comment:
I think it would be better to reset the callback to `null` when unbound to the service in `onServiceDisconnected` for example.

Reply:
Right, done.